config/helm/chart/default/tests/Common/webhook/mutatingwebhookconfiguration_test.yaml

```diff
@@ -67,3 +67,16 @@ tests:
       - equal:
           path: webhooks[1].timeoutSeconds
           value: 13
+  - it: should change failurePolicy
```

Comment:
we could use a test for the default value

Reply:
done in [Add extra test for defaults](https://github.com/Dynatrace/dynatrace-operator/pull/3652/commits/34940c3875268991308afcdc73e133d4caf454e5)